src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -52,6 +56,34 @@ class ComposeUp extends DefaultTask {
         }
     }
 
+    protected void captureContainersOutput() {
+        def t = Executors.defaultThreadFactory().newThread(new Runnable() {
+            @Override
+            void run() {
+                project.exec { ExecSpec e ->
+                    e.commandLine extension.composeCommand('logs', '-f', '--no-color')
+                    e.standardOutput = new OutputStream() {
+                        def buffer = new ArrayList<Byte>()
+                        @Override
+                        void write(int b) throws IOException {
+                            if (b == 10 || b == 13) {

```

Comment:
what is 10 and 13? Because of not knowing that I am unable to finish reviewing this - because I don't know whether the rest is right... :)